spec/sidekiq_adhoc_job/utils/class_inspector_spec.rb

```diff
@@ -8,6 +8,8 @@
   describe "#parameters" do
     it do
       expect(inspector.parameters(:perform)).to eq({
+        :key => [:dryrun],
+        :keyreq => [:type],
```

Comment:
```suggestion
        key: [:dryrun],
        keyreq: [:type],
```

Let's follow the convention of the rest of the key-value declaration.